src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -227,6 +272,80 @@ private void testEditInstructorAction() {
         
         courseEditPage.editInstructor(instructorId, invalidName, "teammates@email.tmt");
         courseEditPage.verifyStatus((new FieldValidator()).getInvalidityInfo(FieldType.PERSON_NAME, invalidName));
+        
+        ______TS("success: test Custom radio button getting other privileges' default values when selected");
+        instructorIndex = 2;
+        courseEditPage.clickEditInstructorLink(instructorIndex);
+        
+        // Tutor
+        courseEditPage.selectRoleForInstructor(instructorIndex, "Tutor");
+        courseEditPage.clickSaveInstructorButton(instructorIndex);
+        courseEditPage.clickEditInstructorLink(instructorIndex);
+        courseEditPage.selectRoleForInstructor(instructorIndex, "Custom");
+        assertFalse(courseEditPage.isPrivilegeCheckboxInPermissionDivChecked(instructorIndex, "canmodifycourse"));
+        assertFalse(courseEditPage.isPrivilegeCheckboxInPermissionDivChecked(instructorIndex, "canmodifyinstructor"));
```

Comment:
Oops, sorry forgot to mention this earlier, but you should probably use the constants from <code>Const.ParamsNames</code> for these strings